libraries/commerce/favorites/actions/toggleFavorites.js

```diff
@@ -59,8 +66,17 @@ export const requestSync = () => (dispatch, getState) => {
  * @param {string} productId Product identifier.
  * @returns {Promise} PipelineRequest dispatch.
  */
-const addFavorites = productId => (dispatch) => {
+const addFavorites = productId => (dispatch, getState) => {
   dispatch(requestAddFavorites(productId));
+
+  const count = getFavoritesCount(getState());
+  if (count >= limit) {
```

Comment:
This is not entirely correct. first the item is added and then it checked to be `>=` but that will cause the error to be triggered one item too early (I have 99, then add one and reach the limit without exceeding it, but the error will still occur).

Reply:
Fixed.